src/nodes/table.ts

```diff
@@ -110,9 +110,16 @@ export function renderNodeToLatex(state: MarkdownSerializerState, node: Node<any
         attrs: { colspan },
       } = cell;
       if (colspan > 1) state.write(`\\multicolumn{${colspan}}{c}{`);
+
       cell.content.forEach((content) => {
         // NOTE: this doesn't work well for multi-paragraphs
+        if (content.type.name === 'equation') {
+          state.write('\\(\\displaystyle ');
```

Comment:
I think it might be worth centralizing this code in the equations component. and that code check if it is in a table cell and render differently.

We can't have equation numbers or labels I don't think in latex inside of tables, for example.